ctms/bin/acoustic_sync.py

```diff
@@ -48,21 +48,28 @@ def main(db, settings):
         update_healthcheck(healthcheck_path)
 
         duration_s = monotonic() - prev
+
         if context["count_total"] == context["batch_limit"]:
             to_sleep = 0
+
         else:
             to_sleep = settings.acoustic_loop_min_secs - duration_s
 
         if context["count_total"] == 0:
             context["trivial"] = True
+
         logger.info(
             "sync_service cycle complete",
             loop_duration_s=round(duration_s, 3),
             loop_sleep_s=round(to_sleep, 3),
             **context
         )
+
+        metric_service.observe_acoustic_sync_loop_duration(round(duration_s, 3))
```

Comment:
Oh, so we're tracking how long a batch takes to sync before we sleep?

Reply:
right, should tell us how to schedule the healthcheck intervals